docs/manuals/auth/index.md

```diff
@@ -12,19 +12,18 @@ To use LDAP authentication, you must configure how Taipy connects to the LDAP se
 - **Config.global_config.ldap_base_dn**: The base DN that the LDAP server uses to search for users authentication.
 - **Config.global_config.secret_key**: The secret key that will be used to generate the access token.
 - **Config.global_config.auth_session_duration**: The period after which the token provided by Taipy will expire.
-- **Config.global_config.auth_enabled**: The field to enable or disable Taipy authentication and authorization. Its default value is True, meaning that the enterprise version, by default, will have authentication and authorization feature enabled.
-- **Config.global_config.authentication_protocol**: The authentication protocol that will be used to authenticate users. Currently Taipy accepts two protocols: "ldap" and "taipy" (for testing purposes only).
+- **Config.global_config.authenticator_protocol**: The authentication protocol that will be used to authenticate users. Currently Taipy accepts three protocols: None, "ldap" and "taipy" (for testing purposes only). By providing the field with None value, we also disable Taipy authentication and authorization feature. And by choosing other protocols ("ldap" or "taipy"), we also enable Taipy authentication and authorization. The default value of this field is None, meaning that authentication and authorization will be disabled by default.
```

Comment:
I would use auth_protocol as well for consistency reasons.
In any case, make sure it is consistent with the code.

Reply:
sure I completely agree with that :D